migration/v0.18.x-v0.19.x/opensearch-migration-clean.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+IFS=$'\n'
```

Comment:
The reason for this? I don't see any arrays being created, but maybe there's something else that uses this.

Reply:
I think I just used one as a template for the others.

Reply:
Right, as long as it works I'm fine with keeping it.